deployments/cloudfoundry/bosh/tile.yml

```diff
@@ -0,0 +1,159 @@
+---
+name: signalfx-monitoring
+icon_file: resources/icon.png
```

Comment:
the icon being used here is the old one, we should update it.

Reply:
Updated to the latest used in the Grafana plugin.